phaudit/7.4/README.md

```diff
@@ -0,0 +1,41 @@
+# Phaudit
+
+- [`7.4`, `latest` (*phaudit/7.4/Dockerfile*)](https://github.com/mykiwi/dockerfiles/tree/master/phaudit/7.4/Dockerfile)
+- [`7.3`, `latest` (*phaudit/7.3/Dockerfile*)](https://github.com/mykiwi/dockerfiles/tree/master/phaudit/7.3/Dockerfile)
```

Comment:
should not be latest
\+ we should update all `README.md` in `phaudit/`

Reply:
done